modules/release-notes/snip-feature-rhidp-3569.adoc

```diff
@@ -0,0 +1,10 @@
+[id="feature-rhidp-3569"]
+= Restarting {product} faster
+
+Before this update, it took a long time for {product-short} to restart because {product-short} bootstraps all dynamic plugins from zero with every restart. 
+
+With this update, {product-short} is using persisted volumes for the dynamic plugins.
+Therefore, {product-short} restarts faster.
+
```

Comment:
Should we link to the product docs from here? It will be _Ch 7 Using the Dynamic Plugins Cache_ in the _Configuring dynamic plugins_ guide.

Reply:
This guide is not merged yet. Add the link once after merge.